src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2720,8 +2720,10 @@ subroutine CNLivewoodTurnover (num_soilp, filter_soilp, &
             if (CNratio_floating .eqv. .true.) then    
                if (livestemc(p) == 0.0_r8) then    
                    ntovr = 0.0_r8    
+                   livestemn_to_deadstemn(p) = 0.0_r8 
                 else    
                    ntovr = ctovr * (livestemn(p) / livestemc(p))   
+                   livestemn_to_deadstemn(p) = ctovr / deadwdcn(ivt(p)) 
                 end if   
 
                 livestemn_to_deadstemn(p) = 0.5_r8 * ntovr   ! assuming 50% goes to deadstemn 
```

Comment:
Doesn't this line override what was done earlier?

Reply:
I guess it would... 
This bottom line should be commented out / removed `livestemn_to_deadstemn(p) = 0.5_r8 * ntovr   ! assuming 50% goes to deadstemn`, as with roots, below